processing/src/test/java/io/druid/query/metadata/SegmentMetadataQueryTest.java

```diff
@@ -100,10 +100,10 @@ public SegmentMetadataQueryTest()
             new ColumnAnalysis(
                 ValueType.STRING.toString(),
                 10881,
-                1,
+                2,
                 null
             )
-        ), 71982
+        ), 73191
```

Comment:
@nishantmonu51 Null dimensions are now stored in the index on disk if the dimension was present in the dimension ordering, the sample has an empty dim called null_column, so the size returned by segment metadata query is larger now